src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -90,7 +88,8 @@ struct sd_dhcp_client {
         uint32_t mtu;
         uint32_t xid;
         usec_t start_time;
-        unsigned attempt;
+        uint64_t attempt;
+        uint64_t max_client_attempt;
```

Comment:
attempts

Dunno, if we have `attempt`, this should be `max_attempts`, without "client" to match.